src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception.html.twig

```diff
@@ -1,7 +1,7 @@
 <div class="block-exception">
     <div class="block-exception-detected clear-fix">
         <div class="support">
-            <a href="http://symfony.com/support">Need support?</a>
+            <a href="http://symfony.com/support">Support</a> | <a href="https://google.com/search?q={{ exception.message|nl2br|format_file_from_text }}" rel="noopener noreferrer" target="_blank">Google Search</a>
```

Comment:
Yes, maybe someone,  who knows Symfony better than me, will implement this filter it in future